buildkite/src/Lib/Cmds.dhall

```diff
@@ -13,8 +13,16 @@ let module =
           \(environment : List Text)
       ->  let Docker =
                 { Type =
-                    { image : Text, extraEnv : List Text, privileged : Bool }
-                , default = { extraEnv = [] : List Text, privileged = False }
+                    { image : Text
+                    , extraEnv : List Text
+                    , privileged : Bool
+                    , useBash : Bool
```

Comment:
This appears to be unused in this PR, and generally orthogonal to the changes. Can you remove this?